### ISSUE TITLE
SSL CACert Location Unknown outside of Linux

### ISSUE THREAD

**Issue by fchorney** [JuliaWeb/LibCURL.jl#59]
Since the LibCURL library is cross compiled on a Linux instance, when it compiles the linux versions it sets the CACert location to the default location on Linux. For the other OS's (Apple, Windows, etc) it leaves the CACert location blank in the build as it does not know where to find them. This causes the other OS's to not have any CACerts to validate SSL/TLS certificates. Currently the only solution would be to supply your own cert when you are using LibCURL.jl

**Comment by fchorney** [comment]
As an example if you run the following code on an apple device:
```julia
using LibCURL

# init a curl handle
curl = curl_easy_init()

# set the URL and request to follow redirects
curl_easy_setopt(curl, CURLOPT_URL, "https://www.google.com/")
curl_easy_setopt(curl, CURLOPT_USE_SSL, CURLUSESSL_ALL)
curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, 2)
curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, 1)
curl_easy_setopt(curl, CURLOPT_VERBOSE, 1)

# execute the query
res = curl_easy_perform(curl)
ok = res == CURLE_OK
println("curl url exec response : ", res)
println("curle_ok: ", ok)

# release handle
curl_easy_cleanup(curl)
```

you will see:
```
*   Trying 172.217.165.4...
* TCP_NODELAY set
* Connected to www.google.com (172.217.165.4) port 443 (#0)
* mbedTLS: Connecting to www.google.com:443
* mbedTLS: Set min SSL version to TLS 1.0
* mbedTLS: Handshake complete, cipher is TLS-ECDHE-ECDSA-WITH-AES-128-GCM-SHA256
* Cert verify failed: BADCERT_NOT_TRUSTED
* Closing connection 0
curl url exec response : 51
curle_ok: false
```

**Comment by omus** [comment]
Broken test added in: https://github.com/JuliaWeb/LibCURL.jl/pull/58

**Comment by omus** [comment]
As of #75 all OS' should now have valid CA certs. Included in the v0.6.0 release.
